src/M6Web/Bundle/LogBridgeBundle/Matcher/Dumper/PhpMatcherDumper.php

```diff
@@ -179,7 +182,7 @@ private function generateMatchList(Configuration $configuration): string
                     }
 
                     $code .= "],\n";
-                } else {
+                } elseif(is_string($config) || is_int($config)) {
                     $code .= sprintf("            '%s' => '%s',\n", $key, $config);
                 }
```

Comment:
I had trouble fixing a PHPStan issue here, just using a comment wasn't enough...
```
 ------ ---------------------------------------------------------------------------------------------- 
  Line   M6Web/Bundle/LogBridgeBundle/Matcher/Dumper/PhpMatcherDumper.php                              
 ------ ---------------------------------------------------------------------------------------------- 
  188    Parameter #3 ...$values of function sprintf expects bool|float|int|string|null, mixed given.  
 ------ ---------------------------------------------------------------------------------------------- 
```
I think this might be ok but I don't exactly know how it should behave... can anyone confirm this ?

Reply:
```suggestion
                } elseif(is_string($config) || is_int($config)) {
                    /** @var bool|float|int|string|null $config */
                    $code .= sprintf("            '%s' => '%s',\n", $key, $config);
                }
```

does not work?

it seems that PHPStan flag the value as `mixed`, if `!is_array`... and he's right, there are a lot of other possible types (e.g. any classes/enums/functions/...)